### PR TITLE
Bump rubocop up to match hound

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

**Comment by Br3nda** [comment]
Noteworthy change here.
There's new cops now, that we aren't matching yet. Hound will tell us about these as we go. So i've turned off rubocop check in travis.

File: lib/tasks/sign_image_tasks.rake [hunk 1]
```diff
@@ -1,7 +1,7 @@
 require 'fileutils'
 
 namespace :sign_images do
-  dimensions_regex = Regexp.new(/(\d{1,4})x(\d{1,4})/)
+  DIMENSIONS_REGEXP = Regexp.new(/(\d{1,4})x(\d{1,4})/)
```
Comment (eoinkelly) [inline lib/tasks/sign_image_tasks.rake]: :elephant: FYI you don't need the `Regexp.new` to create a regex literal - the forward ticks creates a regex literal by itself

Comment (Br3nda) [inline lib/tasks/sign_image_tasks.rake]: so, just `/(\d{1,4})x(\d{1,4})/`
?

Comment (Br3nda) [inline lib/tasks/sign_image_tasks.rake]: i'm 99% sure this rake task doesn't run anywhere anymore

Comment (eoinkelly) [inline lib/tasks/sign_image_tasks.rake]: yep

**Comment by Br3nda** [comment]
"49 fixed issues". says codeclimate

**Pull request merged**